Documentation/network/pod-mac-address.rst

```diff
@@ -0,0 +1,69 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _pod_mac_address:
+
+************************************
+Use a Specific MAC Address for a Pod
+************************************
+
+
+Some applications bind software licenses to network interface MAC addresses.
+Typically, the solution is very complex and requires things like Docker within
+the cluster.
+
+Therefore, cilium provides the ability to specific MAC addresses
+for pods instead of letting the operating system allocate them.
+
+
+
+
+Specific MAC Address
+#####################
+
+The MAC address configured by the annotation described here will be visible
+from within the container on the eth0 interface. Since it is isolated to the
+container it will not collide with any other MAC addresses assigned to other
+pods on the same node.
+
+
+Annotate the pod with ``cni.cilium.io/mac-address`` set to the desired MAC address.
+After adding the annotation to the pod, it appears like so.
+
+.. code-block:: yaml
+
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      annotations:
+        cni.cilium.io/mac-address: e2:9c:30:38:52:61
+      labels:
+        app: busybox
+      name: busybox
+      namespace: default
+
+After the pod runs successfully, we can query the pod's eth0 interface.
```

Comment:
If the user needs to do something, we should instruct them to do it in simple terms. We shouldn't assume that they do something via wording like "after the pod runs successfully".

```suggestion
Deploy the Pod. Cilium will configure the mac address to the first interface in the Pod automatically.
```

Reply:
Updated.